src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml

```diff
@@ -0,0 +1,428 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:ios="http://uno.ui/ios"
+                    xmlns:macos="http://uno.ui/macos"
+                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                    xmlns:not_macos="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:um="using:Uno.Material"
+                    mc:Ignorable="ios macos">
+
+    <x:String x:Key="M3MaterialRevealGlyphPathData">M16,5.5999984C17.699951,5.5999984 19.099976,6.9999918 19.099976,8.6999893 19.099976,10.400001 17.699951,11.799994 16,11.799994 14.299988,11.799994 12.899963,10.400001 12.899963,8.6999893 12.899963,6.9999918 14.299988,5.5999984 16,5.5999984z M16,2.1999893C12.399963,2.1999893 9.5,5.0999984 9.5,8.6999893 9.5,12.299994 12.399963,15.199988 16,15.199988 19.599976,15.199988 22.5,12.299994 22.5,8.6999893 22.5,5.0999984 19.599976,2.1999893 16,2.1999893z M16,0C24.799988,5.2249334E-08 32,8.6999893 32,8.6999893 32,8.6999893 24.799988,17.400001 16,17.400001 7.1999512,17.400001 0,8.6999893 0,8.6999893 0,8.6999893 7.1999512,5.2249334E-08 16,0z</x:String>
+
+    <SolidColorBrush x:Key="M3MaterialDisabledOutlinedPasswordBoxBorderBrush"
+                     Opacity="0.12"
+                     Color="{ThemeResource OnSurfaceColor}" />
+
+    <Style x:Name="M3MaterialRevealButtonStyle"
```

Comment:
x:
```suggestion
    <Style x:Key="M3MaterialRevealButtonStyle"
```

Reply:
Nice catch, the v1 PasswordBox styles also have this issue.. gonna do a pass to see if there are others